src/SDKs/SqlManagement/Management.Sql/Generated/DatabasesOperations.cs

```diff
@@ -4003,7 +4003,7 @@ internal DatabasesOperations(SqlManagementClient client)
             HttpStatusCode _statusCode = _httpResponse.StatusCode;
             cancellationToken.ThrowIfCancellationRequested();
             string _responseContent = null;
-            if ((int)_statusCode != 201 && (int)_statusCode != 202)
+            if ((int)_statusCode != 200 && (int)_statusCode != 202)
```

Comment:
Note to reviewers: Swagger change was https://github.com/Azure/azure-rest-api-specs/pull/1166